### ISSUE TITLE
#-1369043354------------- All page content should be contained by landmarks

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1796]
<strong>PageName:</strong> HomePage  

<strong>Source:</strong>`<div class="sc-gmPhUn jGBoSd">`

<strong>Element Location: </strong>.sc-gmPhUn

<strong>Impact: </strong>moderate

<strong>Failure Summary:</strong> Fix any of the following:
  Some page content is not contained by landmarks

<strong>Tags:</strong> cat.keyboard,best-practice

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/region?application=axeAPI
<strong>Screenshot:<strong>`C:\Automation\Workspace\Velocity\Screenshot\HomePage\-1369043354_All page content should be contained by landmarks.png`
